src/hobbits-plugins/displays/Binary/binary.cpp

```diff
@@ -138,3 +139,6 @@ QPoint Binary::headerOffset(const Parameters &parameters)
                 DisplayHelper::textSize(font, container->frameCount()).width() + margin,
                 DisplayHelper::textSize(font, container->maxFrameWidth()).width() + margin);
 }
+
+
```

Comment:
you should clean up files that are just adding whitespace and comment noise to the pull request